metrics_sdk/lib/opentelemetry/sdk/metrics/export/periodic_exporting_metric_reader.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Export
+        class PeriodicExportingMetricReader < MetricReader
+          # exporter - the push exporter where the metrics are sent to.
+          # exportIntervalMillis - the time interval in milliseconds between two consecutive exports. The default value is 60000 (milliseconds).
+          # exportTimeoutMillis - how long the export can run before it is cancelled. The default value is 30000 (milliseconds).
+          def initialize(exporter, aggregation_temporality: nil, export_interval_millis: 60_000, export_timeout_millis: 30_000)
+            @exporter = exporter
```

Comment:
Superclass also initializes `@exporter`, but we don't call `super` here. The superclass also initializes the `metric_store` - is `collect` going to 💥 ?

Reply:
Remove from this PR.